aocd/get.py

```diff
@@ -21,14 +21,14 @@
 log: Logger = getLogger(__name__)
 
 
-def get_data(
+def get_puzzle(
```

Comment:
Rather than modify the existing `get_data` function, which messes up the git history for these lines, add your new function down at L57 instead and leave this one unchanged.